.github/styles/config/vocabularies/Loft/accept.txt

```diff
@@ -44,7 +44,10 @@ RoleBinding
 [sS]ervice  
 ServiceAccount  
 StatefulSet  
+URL
+vCluster
 vCluster Open Source  
-vCluster Pro  
+vCluster Pro
```

Comment:
We should remove vCluster Open Source and vCluster Pro. 

It is now one product `vCluster` with enterprise features not two products.